oslo_messaging/metrics.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import threading, queue
+import statsd
```

Comment:
This needs to be
```python
import logging
import queue
import threading

import statsd
```
to adhere to OpenStack standards

Reply:
got it